src/anet.c

```diff
@@ -514,6 +515,21 @@ static int anetListen(char *err, int s, struct sockaddr *sa, socklen_t len, int
 
     if (sa->sa_family == AF_LOCAL && perm) chmod(((struct sockaddr_un *)sa)->sun_path, perm);
 
+    if (sa->sa_family == AF_LOCAL && group != NULL) {
+        struct group *grp;
+        if ((grp = getgrnam(group)) == NULL) {
+            anetSetError(err, "getgrnam: %s", strerror(errno));
+            close(s);
+            return ANET_ERR;
+        }
+
+        if (chown(((struct sockaddr_un *)sa)->sun_path, /*owner remains same*/ -1, grp->gr_gid)) {
```

Comment:
```suggestion
        if (chown(((struct sockaddr_un *)sa)->sun_path, /*owner remains same*/ -1, grp->gr_gid) == -1) {
```
Is the documented return value.

Reply:
i don't like putting the comment inline into the code, lets start it in a newline.